src/main/java/seedu/address/logic/commands/MarkPaidCommand.java

```diff
@@ -35,31 +35,41 @@ public class MarkPaidCommand extends Command {
             + "Existing months paid of that person will be overwritten by input to this command.\n"
             + "Parameters: INDEX (must be a positive integer)" + "MONTHSPAID... (yyyy-mm format)\n"
```

Comment:
Update the comment as it's not accurate anymore
Parameters: MARKPAIDTARGET (positive integer, or all)

Reply:
Changed to fit new parameters